### DISCUSSION TITLE
How do I manipulate tree style tab with vimium C?

### DISCUSSION THREAD

**Discussion by glyh** [piroor/treestyletab#3642] [Q&A]
It looks like they have a way to interact with other extensions:

https://github.com/gdh1995/vimium-c/wiki/Send-dynamic-messages-to-other-extensions

But what should I send to tree style tab?

**Comment by irvinm** [comment]
Here is the API and what you can do with it:  https://github.com/piroor/treestyletab/wiki/API-for-other-addons
